priv/repo/migrations/20231220204054_add_override_for_test_groups.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Skate.Repo.Migrations.AddOverrideForTestGroups do
+  use Ecto.Migration
+
+  def change do
+    # excellent_migrations:safety-assured-for-this-file raw_sql_executed
+    execute(
+      "CREATE TYPE test_group_override AS ENUM ('none', 'enabled', 'disabled')",
+      "DROP TYPE test_group_override"
+    )
+
+    alter table(:test_groups) do
+      add(:override, :test_group_override)
```

Comment:
**suggestion:** can we add a default value of `none` and thereby avoid handling null values? Both in terms of the application code and in terms of marking the column itself as non-nullable.

Reply:
[`excellent_migrations` doesn't like adding columns with default values](https://hexdocs.pm/excellent_migrations/readme.html#adding-a-column-with-a-default-value)

I would probably be okay with doing it anyway and making `excellent_migrations` ignore the "error" though, especially since this table is pretty small.

Reply:
That page also says this becomes safe in Postgres 11+, and we're on 15. It's understandable since `excellent_migrations` has no way of knowing what Postgres version we're on, but we should probably just disable that warning somehow.

Reply:
Touché - I had been meaning to ask about that. 

Updated, and added config to disable that check in particular.